### PR TITLE
Vehicle model

### PR DESCRIPTION
**Pull request by domcie99**
Please review my code.

### CODE REVIEW

File: src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Address.java [hunk 1]
```diff
@@ -1,4 +1,6 @@
 package pl.ciesielski.dominik.app.cardealerapp.model;
 
 public class Address {
+    private String city;
```
Comment (jackverone) [inline src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Address.java:4]: Please add getter/setter or Lombok @Data.

File: src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Vehicle.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package pl.ciesielski.dominik.app.cardealerapp.model;
+
+public class Vehicle {
```
Comment (jackverone) [inline src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Vehicle.java:3]: Looks good.

**Review by jackverone — CHANGES_REQUESTED** [review]
Please make changes!

File: src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Address.java (hunk 1 above)
Comment (domcie99) [reply src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Address.java:4]: Ok, will do.

File: src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Vehicle.java (hunk 2 above)
Comment (domcie99) [reply src/main/java/pl/ciesielski/dominik/app/cardealerapp/model/Vehicle.java:3]: Thanks.

**Review by jackverone — APPROVED** [review]
Thanks!

**Pull request merged**